changes/984.feature.md

```diff
@@ -1 +1 @@
-old_x keyword arguments in the event factory now default to None.
+`old_x` keyword arguments in the event factory now default to `None`.
```

Comment:
You know this doesn't refer to anything in code right

Reply:
Again, formatting, but will revert